spec/models/admin_only/reports/payments_covering_year_csv_report_spec.rb

```diff
@@ -39,7 +39,6 @@
       successful_this_dec31 = create(:payment, :successful, start_date: this_dec_31)
       create(:payment, :successful, start_date: next_jan_1)
 
-      create(:payment, :expired, start_date: prev_dec_31)
```

Comment:
This should not be removed.
See my comment in `spec/factories/payments.rb`

We need to have tests to ensure that we can handle (old) HIPS payments that still exist in the system. Likewise with the /spec tests: Don't remove any tests that created / used :expired or :awaiting_payment because we need to ensure that we can still deal with HIPS payments in that are in the system.